solr/core/src/java/org/apache/solr/handler/ReplicationHandler.java

```diff
@@ -1768,13 +1770,13 @@ private static Long readIntervalNs(String interval) {
 
   private static final String EXCEPTION = "exception";
 
-  public static final String MASTER_URL = "masterUrl";
+  public static final String PRIMARY_URL = "primaryUrl";
 
   public static final String FETCH_FROM_LEADER = "fetchFromLeader";
 
-  // in case of TLOG replica, if masterVersion = zero, don't do commit
+  // in case of TLOG replica, if primaryVersion = zero, don't do commit
   // otherwise updates from current tlog won't copied over properly to the new tlog, leading to data loss
-  public static final String SKIP_COMMIT_ON_MASTER_VERSION_ZERO = "skipCommitOnMasterVersionZero";
+  public static final String SKIP_COMMIT_ON_MASTER_VERSION_ZERO = "skipCommitOnPrimaryVersionZero";
```

Comment:
Missed updating the constant name? The value on the other side is used as a parameter between instances, we should be able to send the previous value given a back compat mode (so that old instances can read what new instances are sending)